src/test/java/seedu/address/logic/commands/EditCommandTest.java

```diff
@@ -33,7 +33,7 @@
 import seedu.address.network.NetworkManager;
 import seedu.address.testutil.BookBuilder;
 import seedu.address.testutil.EditDescriptorBuilder;
-
+//@@author 592363789
```

Comment:
Move this to [before `execute_invalidBookIndexUnfilteredList_failure`](https://github.com/CS2103JAN2018-F14-B4/main/pull/111/commits/0dc97cb90b9abe89eb37b1c28b22b4b1d56189a6#diff-b4022e7b13b1c0ba6377165b2cd16dcbR136).